flang/include/flang/Optimizer/Builder/Runtime/Command.h

```diff
@@ -0,0 +1,26 @@
+//===-- Command.h - generate assignment runtime API calls --------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+#define FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+
+namespace mlir {
+class Value;
+class Location;
+} // namespace mlir
+
+namespace fir {
+class FirOpBuilder;
+}
+
+namespace fir::runtime {
+
+mlir::Value genArgumentCount(fir::FirOpBuilder &, mlir::Location);
```

Comment:
Should be documented. (doxygen)

Reply:
Should be `genCommandArgumentCount` for consistency.

Reply:
Done both the doxygen doc and "genCommandArgumentCount". Updated the pull request title for now. Commit message will prob be updated after merging the patches.